corgi/tasks/common.py

```diff
@@ -57,3 +59,17 @@ def get_last_success_for_task(task_name):
     return (
         last_success - timedelta(minutes=30) if last_success else timezone.now() - timedelta(days=3)
     )
+
+
+def _get_redis():
+    return app.broker_connection().default_channel.client
+
+
+def set_task_complete(key: str):
+    conn = _get_redis()
+    conn.set(key, int(True))
```

Comment:
Note: This flag will mean if we ever want to reprocess the task again we'll have to set the associate key back to False or delete all the redis data.

Reply:
celery singleton maybe a good route to investigate though not a 'this sprint' thing - it might be useful to step back a little and think what we want from a queue, then incrementalism.

Reply:
I think celery-singleton is a good idea. I posted another PR now which uses it for fetch_brew_build and it seems to work well. I will close this PR, and open another one which splits out the save_prod_tax call to a new task which is a singleton, which should fix CORGI-152 without the need for the direct flag save in Redis proposed in this PR.